File: Progamming Languages/C#/OOP/exercise_38/Adult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exercise_38
{
    public class Adult:Person
    {
        public Adult(){}

        public Adult(string name, int age, string address, List<Book> listBook):base(name, age, address, listBook){

        }

        public override long moneyPay(){
            return ListBook.Count * 10000;
        }

        public void display(){
            Console.WriteLine("Adult!");
            base.display();
        }
    }
}

File: Progamming Languages/C++/OOP Person/Adult.cpp
#include<iostream>
#include "Person.cpp"
using namespace std;
class Adult : public Person{
    public:
        Adult(){}
        Adult(string name, int age, string add, vector<Book> vb):Person(name, age, add, vb){}
        long moneyPay(){
            return Person::getVB().size()*10000;
        }
        void display(){
            cout<<"Adult!"<<endl;
            Person::display();
        }
};